util/xxh3.h

```diff
@@ -0,0 +1,1633 @@
+// Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Comment:
Can you place the whole facebook header, not just this line here?
```
//  Copyright (c) 2019-present, Facebook, Inc.  All rights reserved.
//  This source code is licensed under both the GPLv2 (found in the
//  COPYING file in the root directory) and Apache 2.0 License
//  (found in the LICENSE.Apache file in the root directory).
```
I do think BSD is a weaker license than the license we use so it doesn't make a difference (though I'm not a lawyer), but it feels nice to be explicit here.

Reply:
I was making it look like xxhash.h: https://github.com/facebook/rocksdb/blob/master/util/xxhash.h

Reply:
I still think it is better to include the full header.

Reply:
I can update all the xxh* and hope the update makes sense. ;)

Reply:
Facebook and Yann own copyright to the file, so it can't go wrong from legal perspective in the worst case.